app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
+    if @micropost.save
+      flash[:success] = t ".success"
+      redirect_to root_url
+    else
+      @feed_items = []
```

Comment:
dòng này để làm gì nhỉ, không có thì được không ?

Reply:
dòng này giúp trong việc gửi micropost không thành công, cần gán cho nó một mảng trống để loại bỏ hoàn toàn nguồn cấp dữ liệu. Ko có cũng ko sao ạ